packages/v4/src/content/design-guidelines/components/alert-group/alert-group.md

```diff
@@ -26,4 +27,12 @@ Use 8px of padding between each alert in an alert group. Alert group alignment o
 
 <img src="./img/alert-group-mobile.png" alt="An example of a centered alert group over a mobile view." width="375" />
 
+## Managing Overflow
```

Comment:
Overflow should be lower case O !

Reply:
Fixed it @mmenestr . Please take another look when you get a chance. Thanks.